docs/crocks/Identity.md

```diff
@@ -0,0 +1,63 @@
+# Identity
+
+`Identity : a -> a`
+
+It's just a function which returns what you pass into it.
+
+## What a crock!
+
+It converts your `non-crock` into a `crock`
+
+```js
+const frozenPizza = Identity('pizza') //Identity pizza
+const microwave = x => 'microwave ' + x
+const dinner = frozenPizza.map(microwave) //Identity "microwave pizza"
+dinner.value() // "microwave pizza"
+```
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`equals`](#equals), [`map`](#map), [`of`](#of), [`sequence`](#sequence), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### of
+
+```js
+crocks.Identity.of([1,2,10])
+crocks.Identity([1,2,10])
+```
+
```

Comment:
Actually a better example to highlight the difference would be `IO`.
You can only construct an IO with a function. BUT `of` will take anything you pass it and not check it at all and give you back an IO that when run, returns whatever it was you passed `of`